xml/System.Net.WebSockets/ValueWebSocketReceiveResult.xml

```diff
@@ -127,8 +127,8 @@
         <ReturnType>System.Net.WebSockets.WebSocketMessageType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the type of the current message.</summary>
+        <value>One of the enumeration values that specifies if the current message is a UTF-8, binary or close message.</value>
```

Comment:
```suggestion
        <value>One of the enumeration values that specifies if the current message is a UTF-8, binary, or close message.</value>
```

Reply:
I didn't know you were supposed to place a comma before a conjunction 
@carlossanlop

Reply:
That's an oxford comma. And it was Maira's suggestion, not mine :smile:

Reply:
All about commas: https://docs.microsoft.com/en-us/style-guide/punctuation/commas 😉

Reply:
All right; that is not a thing in Spanish and I thought English had the same rule.

Reply:
Not in Portuguese either. And it's not a rule you have to follow in English, but we do follow at Microsoft.

There are people who hate Oxford commas and some who love. You can also search for Oxford comma memes.

Reply:
> There are people who hate Oxford commas and some who love. You can also search for Oxford comma memes.

So funny!